content/vault/v1.20.x/content/docs/audit/index.mdx

```diff
@@ -4,228 +4,133 @@ page_title: Audit Devices
 description: Audit devices are mountable devices that log requests and responses in Vault.
 ---
 
-# Audit devices
+# Audit logging
 
-Audit devices are the components in Vault that collectively keep a detailed log of all
-requests to Vault, and their responses. Because every operation with Vault is an API
-request/response, when using a single audit device, the audit log contains _every_ interaction with
-the Vault API, including errors - except for a few paths which do not go via the audit system.
+Audit devices enable the detailed recording of all Vault API requests and responses, except for a small set of [API endpoints](#excepted-api-endpoints). An audit device can be a [file](/vault/docs/audit/file), a [syslog server](/vault/docs/audit/syslog), or a [socket](/vault/docs/audit/socket).
```

Comment:
```suggestion
With a a small set of [exceptions](#excepted-api-endpoints), Vault audit devices enable the detailed recording of all API requests and responses. Audit devices can be [files](/vault/docs/audit/file), a [syslog server](/vault/docs/audit/syslog), or a [socket](/vault/docs/audit/socket).
```
Better to highlight the exceptions first, instead of claiming you can audit "everything" and then contradicting ourselves.